pre-docs/control_repo.md

```diff
@@ -0,0 +1,21 @@
+# The control repository pattern
```

Comment:
I think 'Directory Structure for Plan Development' or somesuch might make more sense, then introducing the concept of a control repo in the first few sentences with a link to the Puppet control repos. As a new user, I wouldn't know what a control repo is or that I should look for this.

Reply:
Ok. I can explore coming from that perspective and add the history as an aside.

Reply:
I kept this section pretty much unchanged, but added a bunch of context around it.